### PR TITLE
Testing new branch

### PR DESCRIPTION
**Pull request by vinnyHero**

### CODE REVIEW

**Comment by vinnyHero** [comment]
@nimeshkverma @VenkateshPamarthi45 please review

File: app/src/main/java/com/goupwards/LoginHomeScreen.java [hunk 1]
```diff
@@ -37,18 +37,21 @@
...
+
+
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         FacebookSdk.sdkInitialize(getApplicationContext());
         setContentView(R.layout.login_new);
         appName = (TextView)findViewById(R.id.title1);
-        Typeface tw_cen_mt = Typeface.createFromAsset(getAssets(),"fonts/tw-cen-mt.ttf");
-        appName.setTypeface(tw_cen_mt);
+//        Typeface tw_cen_mt = Typeface.createFromAsset(getAssets(),"fonts/tw-cen-mt.ttf");
```
Comment (nimeshkverma) [inline app/src/main/java/com/goupwards/LoginHomeScreen.java]: Remove The Comments!

Comment (vinnyHero) [inline app/src/main/java/com/goupwards/LoginHomeScreen.java]: haha

Comment (VenkateshPamarthi45) [inline app/src/main/java/com/goupwards/LoginHomeScreen.java]: vinny create custom text view with type face attached to it.  such that you don't create it in every screen.

sample code : 
`public class ProximaNovaTextViewRegular extends AppCompatTextView {

    private static Typeface proximaNovaRegularTypeface;

    public ProximaNovaTextViewRegular(Context context) {
        super(context);
        init();
    }

    public ProximaNovaTextViewRegular(Context context, AttributeSet attrs) {
        super(context, attrs);
        init();
    }

    public ProximaNovaTextViewRegular(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
        init();
    }

    private void init() {
        if (!isInEditMode()) {
            if(proximaNovaRegularTypeface == null) {
                proximaNovaRegularTypeface = Typeface.createFromAsset(getContext().getAssets(), "fonts/proximanova_regular.ttf");
            }
            setTypeface(proximaNovaRegularTypeface);
        }
    }
}`

**Pull request closed without merging**